server/src/config.ts

```diff
@@ -62,8 +62,29 @@ export function getHostInstance(): string {
   }
 }
 
-export function loadDbConfig(): Knex.Config {
+function getPoolSize(connectionName?: string): number {
+  if (!connectionName) return 0;
```

Comment:
I don't mind `getPoolSize` working as is, but a nit: I think it's a little inconsistent that here `0` is a valid return value...